golangci.yml

```diff
@@ -73,6 +73,7 @@ linters:
     - musttag
     # Deprecated ones:
     - gomnd
+    - execinquery
     # Weird/bad ones:
     - wsl
```

Comment:
Commenting a random line

I like using the following settings as a linter that doesn't match has no sense to me

```yaml
linters-settings:
  nolintlint:
    # Disable to ensure that all nolint directives actually have an effect.
    # Default: false
    allow-unused: true
    # Enable to require an explanation of nonzero length after each nolint directive.
    # Default: false
    require-explanation: true
    # Enable to require nolint directives to mention the specific linter being suppressed.
    # Default: false
    require-specific: true
```

Reply:
I have
```
  nolintlint:
    require-specific: true
```
and I don't think
```
allow-unused: true
```
is good at all - I like my rare nolintlint to be exact

but I should add     require-explanation: true
somehow I though it already did that maybe they changed the default

Reply:
Sorry, I thought the default value was false, I invwetwd it without checking. I indeed expect to reports useless nolint

Reply:
turns out though I didn't have require explanation, getting a bunch of hits
```
periodic/periodic.go:281:33: directive `//nolint:revive` should provide explanation such as `//nolint:revive // this is why` (nolintlint)
type PeriodicRunner interface { //nolint:revive
                                ^
periodic/periodic.go:584:14: directive `//nolint:nestif` should provide explanation such as `//nolint:nestif // this is why` (nolintlint)
	if useQPS { //nolint:nestif
	            ^
```
so if I had it'll break dependabot etc PRs